client/reader/tag-stream/header.jsx

```diff
@@ -76,10 +80,11 @@ class TagStreamHeader extends Component {
 
 		return (
 			<div className={ classes }>
-				<div className="tag-stream__header-title-group">
-					<h1 className="tag-stream__header-title">{ title }</h1>
-					{ description && <h2 className="tag-stream__header-description">{ description }</h2> }
-				</div>
+				<NavigationHeader
+					navigationItems={ [ {} ] }
```

Comment:
nit: Do we need to pass an empty array of one object here? It seems like it would be nicer to just give it a default?